Lombiq.NodeJs.Extensions/CustomExecTasks/ExecWithMutex.cs

```diff
@@ -59,14 +59,14 @@ public class ExecWithMutex : Exec
     public override bool Execute() => MutexAccessToUse switch
     {
         MutexAccess.Shared => new SharedMutex(MutexName, TimeoutSpan).Execute(
-            () => base.Execute(),
-            (message, args) => Log.LogMessage(message, args),
-            (message, args) => Log.LogError(message, args)),
+            base.Execute,
+            Log.LogMessage,
+            Log.LogError),
 
         MutexAccess.Exclusive => new ExclusiveMutex(MutexName, TimeoutSpan).Execute(
-            () => base.Execute(),
-            (message, args) => Log.LogMessage(message, args),
-            (message, args) => Log.LogError(message, args)),
+            base.Execute,
+            Log.LogMessage,
+            Log.LogError),
```

Comment:
This can be formatted more compactly now:

```suggestion
        MutexAccess.Shared => new SharedMutex(MutexName, TimeoutSpan)
            .Execute(base.Execute, Log.LogMessage, Log.LogError),

        MutexAccess.Exclusive => new ExclusiveMutex(MutexName, TimeoutSpan)
            .Execute(base.Execute, Log.LogMessage, Log.LogError),
```

Reply:
Same as https://github.com/Lombiq/Orchard-Training-Demo-Module/pull/114#discussion_r1218389217.